cross/bazarr/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = bazarr
+
+BAZARR_VERSION = 0.8.4.2
+
+PKG_VERS = v$(BAZARR_VERSION)
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_DIST_NAME)
+PKG_DIST_SITE = https://github.com/morpheus65535/bazarr/archive
+PKG_DIR = $(PKG_NAME)-$(BAZARR_VERSION)
```

Comment:
I'm not too sure about the choice I have regarding the `PKG_DIST_FILE` but I'll have a look.

Reply:
As all source files are downloaded into the distrib folder, it is required to use PKG_DIST_NAME to rename the file if the filename (PKG_DIST_FILE) does not contain the package name (i.e. version only) to avoid conflicts with others.

Reply:
Done.